src/index.ts

```diff
@@ -7,7 +7,9 @@ export * from './widgets/icon';
 export * from './widgets/inputfield';
 export * from './widgets/label';
 export * from './widgets/message-popup';
+export * from './widgets/multiple-combobox';
```

Comment:
Have you added multiple-combobox.ts file?

Reply:
multiple-combobox was already added in the previous pull request but it was not included in index